docs/pages/deploy-a-cluster/high-availability.mdx

```diff
@@ -124,6 +124,7 @@ services:
 | `3026` | HTTPS Kubernetes proxy |
 | `3036` | MySQL port |
 | `5432` | Postgres port |
+| `27017` | Mongo port |
```

Comment:
![image](https://github.com/gravitational/teleport/assets/2349184/4cf25c33-c31b-45f7-8a78-23a35515ae1d)


```suggestion
| `27017` | MongoDB port |
```

Reply:
I used that since we refer to it as `Mongo` https://goteleport.com/docs/database-access/reference/configuration/#proxy-configuration